src/keys/KeyManager.ts

```diff
@@ -174,8 +166,7 @@ class KeyManager {
     this.recoveryCode = recoveryCode;
     this.rootCert = rootCert;
     this._dbKey = await this.setupKey(this.dbKeyPath, this.dbKeyBits);
-    this._vaultKey = await this.setupKey(this.vaultKeyPath, this.vaultKeyBits);
-    this.logger.info(`Started ${this.constructor.name}`);
+    this.logger.info('Created Key Manager');
```

Comment:
This is incorrect, replace it with `this.logger.info(`Started ${this.constructor.name}`);`.